src/components/Header.js

```diff
@@ -72,19 +72,17 @@ function Header(){
             window.CrComLib.subscribeState('b', `${index + 331}`, incomingValue => {
                 value = incomingValue;
                 setTempToggleState(value)
-                // console.log('temp state', tempToggleState)
+               
             });
             return value;
```

Comment:
can you clarify what is the expected behavior for `return value;` here? If the code block incomingValue=>{} above is running synchronously, then the function would return `value` from the last iteration of the loop. If the block incomingValue => {} runs asychronously, then the `return value;` would be `undefined`